qa/src/test/resources/application.yml

```
@@ -16,7 +16,9 @@ camunda:
         password: sa
         driver-class-name: org.h2.Driver
   process-test:
-    camunda-docker-image-version: 8.8-SNAPSHOT
+#    camunda-docker-image-version: 8.8-SNAPSHOT
+#    multi-tenancy-enabled: true
+
```

When CPT 8.8 add support
```suggestion
    camunda-docker-image-version: 8.8-SNAPSHOT
```